src/routes/klageskjema/begrunnelse/fullmakt-info.tsx

```diff
@@ -0,0 +1,41 @@
+import AlertStripe from 'nav-frontend-alertstriper';
+import Panel from 'nav-frontend-paneler';
+import { Normaltekst, Element } from 'nav-frontend-typografi';
+import React from 'react';
+import { useContext } from 'react';
+import { AppContext } from '../../../app-context/app-context';
+import { Row } from '../../../styled-components/row';
+import { getFullName } from '../summary/personlige-opplysninger-summary';
+import { foedselsnrFormat, navnFormat } from '../summary/text-formatting';
+import styled from 'styled-components/macro';
+
+const PaddedPanel = styled(Panel)`
+    padding: 20px;
+    max-width: 500px;
+    margin: 0 auto;
+    p:first-of-type {
+        margin-bottom: 10px;
+    }
+`;
+
+const FullmaktInfo = () => {
+    const { klage, fullmaktsgiver } = useContext(AppContext);
+
+    if (klage && klage.fullmaktsgiver && fullmaktsgiver) {
```

Comment:
Hva er forskjellen på `klage.fullmaktsgiver` og `AppContext.fullmaktsgiver`?

Reply:
> Hva er forskjellen på `klage.fullmaktsgiver` og `AppContext.fullmaktsgiver`?

`klage.fullmaktsgiver` er fødselsnummeret til fullmaktsgiver som er satt i skjemaet, `AppContext.fullmaktsgiver` er fullmaktsgiverobjektet vi har fått fra APIet. Fullmaktsgiverobjektet har:

`"folkeregisteridentifikator":{"type":"FNR","identifikasjonsnummer":"03076500192"}`

Tenkte det var enklere å bruke skjemaet sitt `fullmaktsgiver`-nummer (som også er `03076500192`) i stedet for å løpe gjennom objektet og finne `identifikasjonsnummer` der `type: "FNR"`..

Reply:
Aha, skjønner. Det er jeg enig i.